draft-ietf-quic-qpack.md

```diff
@@ -164,14 +164,27 @@ unmodified.
 A string literal without a prefix length noted is an 8-bit prefix string literal
 and follows the definitions in [RFC7541] without modification.
 
-## HEADERS Frames on the Control Stream
+## QPACK Encoder Stream
 
 Table updates can add a table entry, possibly using existing entries to avoid
 transmitting redundant information.  The name can be transmitted as a reference
 to an existing entry in the static or the dynamic table or as a string literal.
 For entries which already exist in the dynamic table, the full entry can also be
 used by reference, creating a duplicate entry.
 
+Each set of encoder insructions is prefaced by its length, encoded as a variable
+length integer with an 8-bit prefix.
```

Comment:
We should probably specify that these need to be complete instructions (i.e. no truncation), and require an error if one of the length-prefixed blocks ends mid-instruction.